common/journald/src/stream.rs

```diff
@@ -208,28 +207,35 @@ impl Reader {
             Err(e) => return Err(JournalError::BadRead(e)),
         };
 
-        let timestamp = match self
+        match self
             .reader
             .timestamp()
-            .map(|timestamp| timestamp.duration_since(UNIX_EPOCH))
+            .ok()
+            .map(|timestamp| SystemTime::now().duration_since(timestamp).ok())
```

Comment:
Clippy will probably complain about the map followed by flatten

Reply:
Did fail, but because of a cargo fmt issue with a comment. Fixed now